portfolio/src/main/webapp/chart-script.js

```diff
@@ -38,6 +37,7 @@ function initSelector() {
 
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
+    checkbox.onclick = drawChart;
```

Comment:
I feel like this function is not very readable. I would suggest something like this. WDYT?
```
function initSelector() {
  const countrySelector = document.getElementById('country-selector');

  let checkboxListHTML =
    `<ul>
      ${checkboxValues.map(value => `
          <li>
            <label>
              <input class="country-checkbox" type="checkbox" onclick="drawChart()">
              <span> ${value} </span>
            </label>
          </li>
        `).join('')
      }
    </ul>`;

  countrySelector.innerHTML += checkboxListHTML;
  checkboxElements = Array.from(countrySelector.getElementsByClassName('country-checkbox'))
}
```

Reply:
yes it looks better.